src/media/js/views/usage.js

```diff
@@ -0,0 +1,14 @@
+define('views/usage',
+    ['l10n', 'log', 'models', 'urls', 'z'],
+    function(l10n, log, models, urls, z) {
```

Comment:
'z' is defined but never used.<br>'urls' is defined but never used.<br>'models' is defined but never used.<br>'log' is defined but never used.